apps/console/src/features/server-configurations/pages/connector-listing-page.tsx

```diff
@@ -215,41 +220,58 @@ export const ConnectorListingPage: FunctionComponent<ConnectorListingPageInterfa
      */
     const renderLoadingPlaceholder = (): ReactElement => {
         const placeholders: ReactElement[] = [];
+        const cardsPerRow: number[] = [ 1, 4, 2, 3 ];
```

Comment:
Better to move these static value to a constant file

Reply:
Since these are not reusable and bound to this function only I think we can keep this here. WDYT?